src/jace/util/jax_helper.py

```diff
@@ -30,6 +30,12 @@
     import numpy as np
 
 
+try:
+    import cupy as cp  # type: ignore[import-not-found]
+except ImportError:
+    cp = None
+
+
```

Comment:
Why is this needed?

Reply:
In case you want to use JaCe one a CPU only machine as the CI.

Reply:
I meant, why is cupy needed here? I can't see where it is used in this module.

Reply:
Before the update of the newest JAX we had to test for the array type, since for cupy we have to call `.toDLPack()` but for numpy it was necessary to call `.__dlpack__()`.
I simply forgot that.